IBPSA/Resources/ReferenceResults/Dymola/IBPSA_Fluid_Geothermal_Borefields_BaseClasses_Boreholes_BaseClasses_Examples_InternalResistancesOneUTube.txt

```diff
@@ -1,14 +1,14 @@
-last-generated=2018-07-20
+last-generated=2019-01-02
 statistics-simulation=
 {
   "linear": " ",
   "nonlinear": " ",
   "number of continuous time states": "2",
   "numerical Jacobians": "0"
 }
-time=[0e+00, 3.6e+03]
-TWal.T=[2.981499938964844e+02, 2.981499938964844e+02]
 TPip1.T=[2.981499938964844e+02, 2.993500061035156e+02, 3.005499877929688e+02, 3.0175e+02, 3.029500122070312e+02, 3.041499938964844e+02, 3.053500061035156e+02, 3.065499877929688e+02, 3.0775e+02, 3.089500122070312e+02, 3.101499938964844e+02, 3.113500061035156e+02, 3.125499877929688e+02, 3.1375e+02, 3.149500122070312e+02, 3.161499938964844e+02, 3.173500061035156e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02, 3.181499938964844e+02]
-TPip2.T=[3.131499938964844e+02, 3.131499938964844e+02]
+TWal.T=[2.981499938964844e+02, 2.981499938964844e+02]
```

Comment:
@mwetter and @JayHuLBL is the different organization of the reference file due to the different Dymola version?? This would be pretty annoying...

Reply:
@damienpicard : Variable names won't change between Dymola versions, but the numeric can slightly change.

Reply:
@mwetter I meant that the variable `time` moved from line 9 to line 13.

Reply:
@damienpicard That could be fixed by sorting the output in BuildingsPy when rewriting the reference results, which seems currently not to be the case.

Reply:
@mwetter this might be useful to avoid overloading the git history with unnecessary changes.